src/libraries/System.Diagnostics.Process/src/System/Diagnostics/ProcessWaitState.Unix.cs

```diff
@@ -550,7 +550,7 @@ private Task WaitForExitAsync(CancellationToken cancellationToken = default(Canc
                         _waitInProgress = null;
                     }
                 }
-            });
```

Comment:
This is the latest instance found on Unix.

Reply:
The change is fine.